internal/namespaces/k8s/v1beta4/custom_kubeconfig_get.go

```diff
@@ -0,0 +1,55 @@
+package k8s
+
+import (
+	"context"
+	"reflect"
+
+	"github.com/scaleway/scaleway-cli/internal/core"
+	k8s "github.com/scaleway/scaleway-sdk-go/api/k8s/v1beta4"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+type k8sKubeconfigGetRequest struct {
```

Comment:
why have k8s as prefix here?

Reply:
to be consistent with the other method/struct